README.md

```diff
@@ -1,2 +1,6 @@
 # WireWorld
-This repository contains information the wiring of all the electrical systems in the casr
+This repository contains information the wiring of all the electrical systems in the car.
+
+# Generator script
+
+To use the generator script, simply call `python diagram_generator.py`. This will combine the various xml files representing sub-diagrams into one large XML file which can be opened as a draw.io notebook. 
```

Comment:
I think there should be more information about how it works. For example, what breaks it (people messing with folder structure, etc.).